server/views/pages/goal/add-note/index.njk

```diff
@@ -42,17 +42,20 @@
         <div class="govuk-form-group">
           <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
 
-          {{ govukTextarea({
-            name: "note",
-            id: "note",
-            value: form.note,
-            label: {
-              text: "Add a note to this goal (optional)",
-              classes: "govuk-label--l",
-              isPageHeading: true
-            },
-            errorMessage: errors | findError('note')
-          }) }}
+          <div class="govuk-form-group">
+            <h1 class="govuk-label-wrapper">
+              <label class="govuk-label govuk-label--l" for="note">
+                Add a note to this goal (optional)
+              </label>
+            </h1>
+            {{ govukWarningText({
+              text: "This note will be seen by the prisoner and other prison staff.",
+              iconFallbackText: "Warning",
+              classes: "govuk-!-margin-top-4"
+            }) }}
+            {{ errors | findError('note') }}
```

Comment:
I don't think this field has any validation on it (optional field), but included the error handler just incase!